Practicum/2_week_solutions.cpp

```diff
@@ -0,0 +1,341 @@
+#include <iostream>
+#include <cmath>
+
+int main()
+{
+    //1
+
+    /*int n;
+    std::cout << "Introduce a number: ";
+    std::cin >> n;
+
+    if (n > 0) {
+        std::cout << "Positive";
+    } else if (n < 0) {
+        std::cout << "Negative";
+    } else {
+        std::cout << "0";
+    }*/
+
+    //2
+
+    /*int n;
+    std::cout << "Introduce a number between 1 and 9: ";
+    std::cin >> n;
+
+    switch (n) {
+    case 1:
+        std::cout << "I";
+        break;
+    case 2:
+        std::cout << "II";
+        break;
+    case 3:
+        std::cout << "III";
+        break;
+    case 4:
+        std::cout << "IV";
+        break;
+    case 5:
+        std::cout << "V";
+        break;
+    case 6:
+        std::cout << "VI";
+        break;
+    case 7:
+        std::cout << "VII";
+        break;
+    case 8:
+        std::cout << "VIII";
+        break;
+    case 9:
+        std::cout << "IX";
+        break;
+    default:
+        break;
+    }*/
+
+    //3
+
+    /*int year;
+    std::cout << "Introduce a number: ";
+    std::cin >> year;
+
+    bool isLeap = false;
+
+    if (year % 4 == 0 && (year % 100 != 0 || year % 400 == 0)) {
+        isLeap = true;
+    }
+
+    if (isLeap) {
+        std::cout << "Leap year";
+    } else std::cout << "No leap year";*/
+
+    //4
+
+    /*int day;
+    std::cout << "Introduce a number (1 - 7): ";
+    std::cin >> day;
+
+    switch (day) {
+    case 1:
+        std::cout << "Monday";
+        break;
+    case 2:
+        std::cout << "Tuesday";
+        break;
+    case 3:
+        std::cout << "Wednesday";
+        break;
+    case 4:
+        std::cout << "Thursday";
+        break;
+    case 5:
+        std::cout << "Friday";
+        break;
+    case 6:
+        std::cout << "Saturday";
+        break;
+    case 7:
+        std::cout << "Sunday";
+        break;
+
+    default:
+        std::cout << "Try again";
+        break;
+    }*/
+
+    //5
+    
+    /*int n1, n2;
+
+    std::cout << "Introduce 2 numbers: ";
+    std::cin >> n1 >> n2;
+
+    int biggerNum = (n1 > n2) ? n1 : n2;
+
+    std::cout << biggerNum;*/
+
+    //6
+    
+    // char letter = ' ';
+    // std::cout << "Introduce a letter: ";
+    // std::cin >> letter;
+
+    // if ((letter >= 'a' && letter <= 'z') || (letter >= 'A' && letter <= 'Z')) {
+    //     if (letter >= 'a' && letter <= 'z') {
```

Comment:
Here it's better not to use "magic numbers" like 32. You can capitalise it with ---> letter -'a' +'A'. And make it lower with -'A' +'a'. Only thing to remember is that the asci code of the capitals are before the small letters.